README.md

```diff
@@ -1,3 +1,65 @@
+# Notice 
+---
+**This project is deprecated and is no longer supported or maintained.**
+
+All tests and checks made by this project were moved to the project [operator-framwork/api](https://github.com/operator-framework/api) and specifically to the validator [OperatorHub](https://github.com/operator-framework/api/blob/v0.10.7/pkg/validation/internal/operatorhub.go). Note that any new requirement has been addresssed to the [operator-framwork/api](https://github.com/operator-framework/api).
```

Comment:
This really only calls out the validation portion of courier, which wasn't really the core use of the project -- it was used as a way to package and push manifests to an AppRegistry endpoint. If we're going to give users specific steps for validation, we should probably at least mention that the AppRegistry packaging format is no longer supported in the project and point them to the bundle format and the Operator Registry at the same time.

Reply:
@kevinrizza thank you for the review. Please, feel free to re-check this one.